### PR TITLE
[Improve][Core] Add test case to check deprecated `result_table_name` and `source_table_name`

### PR DESCRIPTION
**Pull request by Hisoka-X**
### Purpose of this pull request
In #8072 we removed `result_table_name` and `source_table_name`, this PR add test case to check deprecated `result_table_name` and `source_table_name`

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
add new test.

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
cc @hailin0

File: seatunnel-dist/src/test/java/org/apache/seatunnel/api/file/AllFileSpecificationCheckTest.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        List<String> fileTypesCanNotRead =
+                Arrays.asList("parquet", "orc", "xlsx", "xls", "png", "jar", "lzo", "zip", "ico");
+        List<String> fileCanNotRead =
+                Arrays.asList(
+                        "seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/encoding/gbk.json",
+                        "seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/encoding/gbk.xml",
+                        "seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/encoding/gbk_use_attr_format.xml",
+                        "seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/encoding/gbk.txt",
+                        "seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/json/e2e_gbk.json",
+                        "seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/text/e2e_gbk.txt");
```
Comment (hailin0) [inline seatunnel-dist/src/test/java/org/apache/seatunnel/api/file/AllFileSpecificationCheckTest.java:57]: why？

Comment (Hisoka-X) [reply seatunnel-dist/src/test/java/org/apache/seatunnel/api/file/AllFileSpecificationCheckTest.java:57]: Can not read by UTF-8 encode. 
image

**Review by hailin0 — APPROVED** [review]

**Review by Carl-Zhou-CN — APPROVED** [review]
+1

**Pull request merged**